### PR TITLE
feat: Icons - November 2023 Update

### PR DESCRIPTION
**Pull request by solygambas**
Here is the November update for icons, with 24 new icons:

1. BabyFurniture.svg
2. BikeSize.svg
3. Bluetooth.svg
4. Book.svg
5. Collectible.svg
6. ComputerAccessory.svg
7. Donation.svg
8. EngineStart.svg
9. Gps.svg
10. Headlight.svg
11. KeylessAccess.svg
12. MultimediaSystem.svg
13. ParkAssist.svg
14. ParkingSensor.svg
15. PetAccessory.svg
16. RainSensor.svg
17. RearviewCamera.svg
18. RearviewMirror.svg
19. Rental.svg
20. Seat.svg
21. Speaker.svg
22. SpeedIndicator.svg
23. Sunroof.svg
24. WheelRim.svg

Please note that we'll have a separate PR to remove duplicates from our icon set (breaking change, see #71).

### CODE REVIEW

File: assets/spark/icons/Donation.svg
Comment (SimonMarquis) [inline assets/spark/icons/Donation.svg:1]: This one looks different from the others (no padding, larger stroke), is it expected?

File: assets/spark/icons/Rental.svg
Comment (SimonMarquis) [inline assets/spark/icons/Rental.svg:1]: This one looks different from the others (no padding, larger stroke), is it expected?

File: assets/spark/icons/Donation.svg
Comment (solygambas) [reply assets/spark/icons/Donation.svg:1]: Thank you, Simon. This icon falls under the `Categories` entries, whereas the others serve as criteria icons. Yaara and Philip are currently redefining our icon set for more consistency. In the meantime, we'll maintain this distinction.

Comment (SimonMarquis) [reply assets/spark/icons/Donation.svg:1]: ACK thanks for the confirmation :)

**Pull request merged**